assets/css/bootstrap/_variable_overrides.scss

```diff
@@ -12,7 +12,7 @@ $border-color: new_tokens.$gray-400;
 $primary: new_tokens.$eggplant-500;
 $secondary: new_tokens.$gray-500;
 $info: semantic.$info;
-$ui-alert: semantic.$ui-alert;
+$ui-alert: new_tokens.$ui-alert;
```

Comment:
If you update it in `definitions.scss`, you don't need to update it anywhere else because `semantic` is `definitions.scss`

Reply:
Ohhhh my goodness I feel so silly I didn't see that. Yep!